### PR TITLE
Validate OOF

### PR DESCRIPTION
**Pull request by dberget**
# Validate [oof](https://solscan.io/token/FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: [https://twitter.com/oof_solana/status/1734630003973775836?s=20](https://twitter.com/oof_solana/status/1734630003973775836?s=20)
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/oof-2
## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by Stuffalot** [comment]
Mint address: FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8
Symbol: oof
Token Decimals: 4
Token icon: https://ih0.redbubble.net/image.1543649450.5330/raf,360x360,075,t,fafafa:ca443f4786.jpg
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by startom1** [comment]
Mint address: FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8
Symbol: oof
Token Decimals: 4
Token icon: https://ih0.redbubble.net/image.1543649450.5330/raf,360x360,075,t,fafafa:ca443f4786.jpg
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Ox2c** [comment]
Mint address: FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8
Symbol: oof
Token Decimals: 4
Token icon: https://ih0.redbubble.net/image.1543649450.5330/raf,360x360,075,t,fafafa:ca443f4786.jpg
On-chain metadata matches
No duplication on Jupiter's strict list
Oof!

**Comment by pandaiolo** [comment]
Mint address: FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8
Symbol: oof
Token Decimals: 4
Token icon: https://ih0.redbubble.net/image.1543649450.5330/raf,360x360,075,t,fafafa:ca443f4786.jpg
On-chain metadata matches
No duplication on Jupiter's strict list
Oof!

**Comment by turtlejooce** [comment]
Mint address: FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8
Symbol: oof
Token Decimals: 4
Token icon: https://ih0.redbubble.net/image.1543649450.5330/raf,360x360,075,t,fafafa:ca443f4786.jpg
On-chain metadata matches
No duplication on Jupiter's strict list
Oof!

**Review by mathlete76 — COMMENTED** [review]
Valid.

File: validated-tokens.csv [hunk 1]
```diff
@@ -1055,3 +1055,5 @@ PONKE,PONKE,5z3EqYQo9HiCEs3R84RCDMu2n7anpDMxRhdK8PSWmrRC,9,https://i.imgur.com/q
 CATWIFHAT,CIF,G3vWvAaXPHCnncnyAbq6yBRXqfRtEV3h7vExzasZeT6g,6,https://i.postimg.cc/hGBgdxxj/Cif200x200.png,true
 AI,AI,99ouK5YUK3JPGCPX9joNtHsMU7NPpU7w91JN4kdQ97po,9,https://gateway.irys.xyz/HnDwG54RZvJhTD2PX-mmrD2jwC9MIdxKjW2hL3M8jQc,true
 catwifhat,$CWIF,7atgF8KQo4wJrD5ATGX7t1V2zVvykPJbFfNeVf1icFv1,2,https://i.postimg.cc/d1QD417z/200x200logo-copy.jpg,true
+OOF,oof,FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8,4,https://i.imgur.com/7at6uGb.jpeg,true
```
Comment (Timoon21) [inline validated-tokens.csv:1058]:
```suggestion
oof,oof,FSM8TmWxGFUjM7Rs6Wt9DHx3ZQAMX6vC4bN8r5PjSvE8,4,https://i.imgur.com/7at6uGb.jpeg,true
```

**Review by Timoon21 — DISMISSED** [review]

**Comment by Jymfletzfur** [comment]
insufficient volume

**Pull request closed without merging**